WordPress/Classes/ViewRelated/Reader/Detail/Views/ReaderDetailToolbar.swift

```diff
@@ -390,3 +486,116 @@ class ReaderDetailToolbar: UIView, NibLoadable {
         reblogButton.accessibilityTraits = UIAccessibilityTraits.button
     }
 }
+
+// MARK: - Private Helpers
+
+private extension ReaderDetailToolbar {
+
+    struct Constants {
+        static let buttonContentInsets = NSDirectionalEdgeInsets(top: 2.0, leading: 0, bottom: 0, trailing: 0)
+        static let buttonImagePadding: CGFloat = 4.0
+
+        static var likeImage: UIImage? {
+            if FeatureFlag.readerImprovements.enabled {
+                // reduce gridicon images to 20x20 since they don't have intrinsic padding.
+                return UIImage(named: "icon-reader-like")?
+                    .resizedImage(WPStyleGuide.Detail.actionBarIconSize, interpolationQuality: .default)
+                    .withRenderingMode(.alwaysTemplate)
+            }
+            return UIImage(named: "icon-reader-like")
+        }
+
+        static var likedImage: UIImage? {
+            if FeatureFlag.readerImprovements.enabled {
+                // reduce gridicon images to 20x20 since they don't have intrinsic padding.
+                return UIImage(named: "icon-reader-liked")?
+                    .resizedImage(WPStyleGuide.Detail.actionBarIconSize, interpolationQuality: .default)
+                    .withRenderingMode(.alwaysTemplate)
+            }
+            return UIImage(named: "icon-reader-liked")
+        }
+
+        static let commentImage = UIImage(named: "icon-reader-comment-outline")?
+            .imageFlippedForRightToLeftLayoutDirection()
+            .resizedImage(WPStyleGuide.Detail.actionBarIconSize, interpolationQuality: .high)
+            .withRenderingMode(.alwaysTemplate)
+
+        static let commentSelectedImage = UIImage(named: "icon-reader-comment-outline-highlighted")?
+            .imageFlippedForRightToLeftLayoutDirection()
+            .resizedImage(WPStyleGuide.Detail.actionBarIconSize, interpolationQuality: .high)
+            .withRenderingMode(.alwaysTemplate)
+
+        // MARK: Strings
+
+        static let savedButtonAccessibilityLabel = NSLocalizedString(
+            "reader.detail.toolbar.saved.button.a11y.label",
+            value: "Saved Post",
+            comment: "Accessibility label for the 'Save Post' button when a post has been saved."
+        )
+
+        static let savedButtonHint = NSLocalizedString(
+            "reader.detail.toolbar.saved.button.a11y.hint",
+            value: "Remove this post from my saved posts.",
```

Comment:
[Nitpick]: [Apple recommends](https://developer.apple.com/library/archive/documentation/UserExperience/Conceptual/iPhoneAccessibility/Making_Application_Accessible/Making_Application_Accessible.html#//apple_ref/doc/uid/TP40008785-CH102-SW5) using the third-person singular declarative form of verbs for the hints:

> Be sure to use the third-person singular declarative form of a verb, such as “Plays,” and not the imperative, such as “Play.” You want to avoid using the imperative, because using it can make the hint sound like a command. For example, you don’t want to tell users to “Play the song”; instead, you want to tell users that tapping the element “Plays the song.”

```suggestion
            value: "Removes this post from my saved posts.",
```

Reply:
Thanks! This is now addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/21763/commits/b6a29ac6d13ab37200b3e647e243caed2a0949ff.